README.md

```diff
@@ -18,6 +18,8 @@ The `k8s-pvc-tagger` watches for new PersistentVolumeClaims and when new AWS EBS
 
 `--allow-all-tags` - Allow all tags to be set via the PVC; even those used by the EBS/EFS controllers. Use with caution!
 
+`--propate-to-pv` - Allow to propagate labels and annotations from PVC to PV. Adding computed tags as annotation with prefix: `k8s-pvc-tagger/tags/`
```

Comment:
What if we used `--add-metadata-to-pv`? Would that work? I'm horrible at naming too.